specs/vep-2420-getting-started-with-my-data/README.md

```diff
@@ -0,0 +1,198 @@
+
+# VEP-2420: VDK Getting Started with My Data
+
+* **Author(s):** Paul Murphy (paulm2@vmware.com), Antoni Ivanov (aivanov@vmware.com)
+<!-- * **Status:** draft | implementable | implemented | rejected | withdrawn | replaced -->
+* **Status:** draft
+
+
+- [VEP-2420: VDK Getting Started with My Data](#vep-2420-getting-started-with-my-data)
+  - [Summary](#summary)
+  - [Glossary](#glossary)
+  - [Motivation](#motivation)
+    - [Poor assistance around configuration options](#Poor-assistance-around-configuration-options)
+    - [Layers of abstraction aren't clear](#Layers-of-abstraction-are-not-clear)
+    - [Poor documentation around secrets/properties](#Poor-documentation-around-secrets-and-properties)
+    - [Not trivial to run in the IDE](#Not-trivial-to-run-in-the-IDE)
+    - [Reliant on env variables](#Reliant-on-env-variables)
+  - [Requirements and goals](#requirements-and-goals)
+  - [High-level design](#high-level-design)
+  - [API design](#api-design)
+  - [Detailed design](#detailed-design)
+  - [Implementation stories](#implementation-stories)
+  - [Alternatives](#alternatives)
+
+## Summary
+
+Users struggle to get started with their own data in VDK.
```

Comment:
This is the most user facing part of the document. THe rest of the document is really meant for the dev team. 

As teh template says: 

```
user-focused
documentation such as release notes or a (customer facing) development roadmap.
The tone and content of the `Summary` section should be
useful for a wide audience.
```

That's why I recommend leaving that for later.

Reply:
I added work in progress to the header. 
I will leave what I have written in?